lightkurve/targetpixelfile.py

```diff
@@ -41,6 +44,110 @@ def __init__(self, path, quality_bitmask='default', targetid=None, **kwargs):
         self.quality_mask = self._quality_mask(quality_bitmask)
         self.targetid = targetid
 
+
+    def get_sources(self, catalog=None, magnitude_limit=18, dist_tolerance=3):
```

Comment:
dist_tolerance is not a very good name. Perhaps edge_tolerance would be more descriptive?